lib/google_maps_geocoder/google_maps_geocoder.rb

```diff
@@ -3,6 +3,8 @@
 require 'net/http'
 require 'rack'
 
+require 'google_maps_geocoder/google_maps_geocoder_error_handler'
```

Comment:
And go ahead and keep it all in one class file. You can disable the Rubocop warning by adding this before the class declaration:

```ruby
# rubocop:disable Metrics/ClassLength
class GoogleMapsGeocoder
```